scripts/migrate-env-for-v0.2.8.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+set -e 
+
+echo "Start Migrating Process"
+echo "-----------------------"
+echo "Backup env and docker-compose file"
+
+cp .env .env_backup 
+cp docker-compose.yml docker-compose-backup.yml
+
+echo "Finish backup for 2 files"
+
+# To new docker-compose config in v0.2.8 
+# Following docs here https://docs.roninchain.com/validators/setup/mainnet/run-combined
+
+# - LISTENERS__RONIN__RPCURL=${RPC_ENDPOINT}
+# - LISTENERS__RONIN__SECRET__BRIDGEOPERATOR__PLAINPRIVATEKEY=${LISTENERS__RONIN__SECRET__BRIDGEOPERATOR__PLAINPRIVATEKEY}
+# - RONIN_LEGACY_BRIDGE_OPERATOR_KEY=${LEGACY_BRIDGE_OPERATOR_PRIVATE_KEY}
+# - RONIN_BRIDGE_VOTER_KEY=${BRIDGE_VOTER_PRIVATE_KEY}
+# - RONIN_RELAYER_KEY=${RONIN_RELAYER_KEY}
+# - LISTENERS__ETHEREUM__RPCURL=${LISTENERS__ETHEREUM__RPCURL}
+# - ETHEREUM_VALIDATOR_KEY=${ETHEREUM_VALIDATOR_KEY}
+# - ETHEREUM_RELAYER_KEY=${ETHEREUM_RELAYER_KEY}
+# - DATABASE__HOST=db
+# - DATABASE__DBNAME=${DATABASE__DBNAME}
+# - DATABASE__PORT=5432
+# - DATABASE__USER=${DATABASE__USER}
+# - DATABASE__PASSWORD=${DATABASE__PASSWORD}
+# - VERBOSITY=${VERBOSITY}
+# - CONFIG_PATH=${CONFIG_PATH}
+# - LISTENERS__RONIN__TASKINTERVAL=${LISTENERS__RONIN__TASKINTERVAL}
+# - LISTENERS__RONIN__TRANSACTIONCHECKPERIOD=${LISTENERS__RONIN__TRANSACTIONCHECKPERIOD}
+# - LISTENERS__RONIN__MAXPROCESSINGTASKS=${LISTENERS__RONIN__MAXPROCESSINGTASKS}
+# - LISTENERS__ETHEREUM__GETLOGSBATCHSIZE=${LISTENERS__ETHEREUM__GETLOGSBATCHSIZE}
+# - LISTENERS__RONIN__GASLIMITBUMPRATIO=${LISTENERS__RONIN__GASLIMITBUMPRATIO}
+# - LISTENERS__RONIN__STATS__NODE=${LISTENERS__RONIN__STATS__NODE}
+# - LISTENERS__RONIN__STATS__HOST=${LISTENERS__RONIN__STATS__HOST}
+# - LISTENERS__RONIN__STATS__SECRET=${LISTENERS__RONIN__STATS__SECRET}
+
+echo "-------------------"
+echo "Start for replacing Ronin and Etherum Endpoint config"
+sed -i -e 's/RPC_ENDPOINT/LISTENERS__RONIN__RPCURL/g' .env
+sed -i -e 's/BRIDGE_OPERATOR_PRIVATE_KEY/LISTENERS__RONIN__SECRET__BRIDGEOPERATOR__PLAINPRIVATEKEY/g' .env
+sed -i -e 's/ETHEREUM_ENDPOINT/LISTENERS__ETHEREUM__RPCURL/g' .env
+
+sed -i -e 's/RONIN_RPC/LISTENERS__RONIN__RPCURL/g' docker-compose.yml
```

Comment:
The current documentation shows `- RONIN_RPC=${RPC_ENDPOINT}`
So we must
- sed -i -e 's/RONIN_RPC/LISTENERS__RONIN__RPCURL/g' docker-compose.yml
- sed -i -e 's/RPC_ENDPOINT/LISTENERS__RONIN__RPCURL/g' docker-compose.yml